doc/lsst.pipe.base/creating-a-pipeline.rst

```diff
@@ -0,0 +1,448 @@
+labels that are not imported (either because they are excluded, or they are
+not part of the include list). If any omitted label appears as part of a
+subset, then the subset subset definition is not imported.
+
+The order that `Pipelines` are listed in the ``imports`` section is not
+important. Another thing to note is that declared labels must be unique
+amongst all inherited `Pipelines`.
+
+Once one or more pipelines is imported the ``PipelineTask``\ s defined in
+the ``tasks`` section are considered. If any new ``labels`` are declared they
+simply extend the total `Pipeline`.
+
+If a ``label`` declared in the the ``tasks`` section was declared in one of
+the imported ``Pipelines``, one of two things happen. If the label is
+associated with the same `PipelineTask` that was declared in the imported 
+pipeline, this definition will be extended. This means that any configs
+declared in the imported `Pipeline` will be merged with configs declared in
+the current `Pipeline` with the current declaration taking config precedence.
+This behavior allows tasks to be extended in the current `Pipeline`.
+
+If the ``label`` declared in the current `Pipeline` is associated with a
+different `PipelineTask` than that of the imported declaration, then the
+label with be considered re-declared and the declaration in the current
+`Pipeline` will be used when processing the complete pipeline. The
+declaration defined in the imported `Pipeline` dropped.
+
+
+-------------------------------------
+obs\_* package overrides for Pipelines
+-------------------------------------
+
+Pipeline files support automatically loading configuration files defined in
+obs packages though a top level key named `instrument`. The value associated
+with this key is a string representing the fully qualified class name of the
+python camera object. For instance for ``obs_subaru`` this would look like:
+
+.. code-block:: yaml
+
+  instrument: lsst.obs.subaru.HyperSuprimeCam
+
+The ``instrument`` is available to all `Pipelines`, but by conventions obs\_*
+packages typically will additionally make `Pipelines` that are customized for
+the instrument they represent. This includes relevant configs, `PipelineTask`
+(re)declarations, instrument label, etc. These pipelines can be found inside
+a directory named `pipelines` that lives at the root of each obs\_ package.
+
+These `Pipeline`\ s enable you to run a `Pipeline` that is configured for the
+desired camera, or to import from
+
+.. _pipeline-running-intro:
+
+---------------------------------
+Commandline options for running Pipelines
+---------------------------------
+This section is not intended to serve as a tutorial for processing data from
+the command line, for that refer to `lsst.ctrl.mpexec` or `lsst.ctrl.bps`.
+However, in any tool in which a `Pipeline` can be specified, specific sytax
+maybe used when specifying the pipeline which will control how the `Pipeline`
+will be loaded.
```

Comment:
You have completely lost me here...to the point where I'm not sure I can comment on the rest of the section.  Is it really appropriate for this guide?  If so, it's going to require more clarity and probably some explicit examples.